plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -402,7 +405,8 @@ def run_inspection(file_filter, options, do_extract, file_store_tmp, file_store_
       if !local_filesystems.empty?
         # force all mount points to be non-leave directories (find is called with -xdev)
         local_filesystems.each do |mp|
-          dirs[mp] = true if dirs.has_key?(mp)
+          path = mp.sub(/^\//, "")
```

Comment:
local_filesystems consist the local ones and btrfs sub volumes. Isn't this a problem when they are different in the beginning?